src/helm/config/model_deployments.yaml

```diff
@@ -104,7 +104,29 @@ model_deployments:
       class_name: "helm.benchmark.window_services.image_generation.clip_window_service.CLIPWindowService"
 
 
-  # Amazon
+  # Amazon nova models
+
+  - name: amazon.nova-pro-v1:0
```

Comment:
Please replace `amazon.nova-pro-v1:0` with `amazon/nova-pro-v1:0` to follow HELM model name conventions.

This means that in `convert_request_to_raw_request()` you will also have to do something like `request.model.replace("/", ".")`.